core/controllers/base_test.py

```diff
@@ -211,6 +211,23 @@ def test_requests_for_invalid_paths(self):
 
         self.delete_json('/library/data', expected_status_int=404)
 
+    def test_show_maintenance_mode_when_enabled(self):
+        swap_maintenace_mode = self.swap(
+            feconf, 'ENABLE_MAINTENANCE_MODE', True)
+        with swap_maintenace_mode:
+            response = (
+                self.get_html_response('/library', expected_status_int=503))
+            self.assertIn('htmlWebpackPlugin.options.statusCode', response.body)
```

Comment:
Should you be checking for the existence of "503" too, somewhere? This assertion doesn't really seem to be testing anything related to the 503 itself.

Reply:
Also perhaps add an assertNotIn that corresponds to line 229 below.